index.bs

```diff
@@ -666,16 +677,14 @@ never exceed the [=sampling frequency=] for the given [=sensor type=].
 ## Conditions to expose sensor readings ## {#concepts-can-expose-sensor-readings}
 
 The user agent must verify that all [=mandatory conditions=] are satisfied to ensure it
-<dfn>can expose sensor readings</dfn>.
+<dfn>can expose sensor readings</dfn> to an [=active document=].
 
 The <dfn>mandatory conditions</dfn> are the following:
- - [=steps to determine the visibility state|Visibility state=] for an
-   [=active document=] of [=platform sensor=]'s associated [=browsing context=]
-   is "visible".
- - [=Currently focused area=] belongs to an [=active document=] of [=platform sensor=]'s associated
-   [=browsing context=] or to an [=active document=] of a [=nested browsing context=] whose
-   [=active document=]'s [=origin=] is the [=same origin-domain=] as the
-   [=top-level browsing context=]'s [=active document=]
+ - [=document visibility state|Visibility state=] of the document is "visible".
+ - The document is [=allowed to use=] all the [=policy-controlled features=] associated
+   with the given [=sensor type=].
+ - [=Currently focused area=] belongs to a document whose origin is [=same origin-domain=]
+   with the origin of the given document.
```

Comment:
of the given active document? Would be easier to connect to initial statement.

Reply:
done